packages/map_editor/forms/new_tag_object.py

```diff
@@ -11,14 +11,22 @@ def __init__(self, tags):
 
     def dialog_accept(self):
         tag_type = self.combo_type.currentText()
-        tag_id =  int(self.combo_id.currentText())
-        self.apriltag_added.emit([dict(kind="apriltag_300",pos=(1.0, 1.0), rotate=0, height=1,
+        tag_id = int(self.lineEdit.text())
```

Comment:
https://stackoverflow.com/questions/15829782/how-to-restrict-user-input-in-qlineedit-in-pyqt